packages/runtime/types/index.d.ts

```diff
@@ -0,0 +1,29 @@
+// TypeScript Version: 3.5
```

Comment:
This is a minor release higher than the others `Omit` was an addition in TypeScript 3.5